### PR TITLE
System test fix: now all tests pass (not in travis)

### LINKED ISSUE #91: Make the test pass and run some at travis
System tests are still not passing and some of them even behaving in a non deterministic way. Check and fix it.

### PR DESCRIPTION
**Pull request by raticate**
fixes #91 
[travis site](https://travis-ci.org/niclabs/tchsm-libdtc/builds)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,8 +12,8 @@ install:
     - make dtc
     - make
     - make system_test
-    - sudo apt-get install -y python3-pip
     - sudo python3 -m pip install -r ../tests/system_test/requirements.txt
 
 script:
     - make check
+    - python3 ../tests/system_test/system_test.py --debug -v -r 'DTC' .
```
Comment (fmontoto) [inline .travis.yml]: Don't run all DTC tests at travis. Run one or two and one or two of PKCS11

Comment (raticate) [inline .travis.yml]: changed that, but now a test failed in travis

**Comment by fmontoto** [comment]
Are we merging this @raticate ?

**Comment by raticate** [comment]
@fmontoto ping

**Comment by fmontoto** [comment]
We're leaving travis out from system testing for now...

**Pull request merged**